iothub/device/src/ClientPropertyCollection.cs

```diff
@@ -87,19 +86,19 @@ public void AddRootProperties(IDictionary<string, object> properties)
         /// <inheritdoc path="/summary" cref="AddInternal(IDictionary{string, object}, string, bool)" />
         /// <inheritdoc path="/remarks" cref="AddOrUpdateComponentProperties(string, IDictionary{string, object})" />
         /// <inheritdoc path="/seealso" cref="AddInternal(IDictionary{string, object}, string, bool)" />
-        /// <inheritdoc path="/exception['ArgumentNullException']" cref="AddRootProperty(string, object)" />
         /// <param name="propertyName">The name of the property to add or update.</param>
         /// <param name="propertyValue">The value of the property to add or update.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="propertyName"/> is <c>null</c>.</exception>
```

Comment:
need to inherit only 1 exception under `AddRootProperty`, so we'll need to specify it explicitly instead.